release-notes/opensearch-release-notes-2.0.0-rc1.md

```diff
@@ -0,0 +1,606 @@
+# OpenSearch and Dashboards 2.0.0-rc1 Release Notes
+
+## Release Highlights
```

Comment:
Some release highlights we should also include are:
* Document level alerting
* Lucene 9
* geo map tile generation pipeline https://github.com/opensearch-project/maps/issues/2
* Term lookup query support in document level security https://github.com/opensearch-project/security/issues/1508

Reply:
@elfisher Any wording suggestion for these highlights? The highlight section wasn't included in the plugin release notes.

Reply:
@CEHENKLE @davidlago Can you help us on the release highlights?

Reply:
Please use the following.

- Document level alerting allows users to create monitors that can generate alerts per document.
- Lucene 9 is now used in OpenSearch
- The Geo Map Tiles in OpenSearch Dashboards are updated and now have a pipeline to update them more frequently.
- Document level security now supports term lookup queries.

Reply:
Added these to the release highlights.